### PR TITLE
Startup Influx Alongside App

### PR DESCRIPTION
**Pull request by tlelievre26**
## What is this PR introducing?

Adds code to automatically start up the local InfluxDB instance when the app launches. The system now does not let the user start a study session until a connection to Influx is established, and adds a settings button where the user can input the path to their local installation of Influx along with their API key.

## What components of our system does this change impact? Which core feature is this relevant to?

Database, start screen

## Pre-commit to-do list

- [x] Does this PR only contain one feature that cannot be broken up into smaller components?
- [ ] Have you written unit/integration tests for this feature?
- [x] Have you run the linter over your code for this change?
- [ ] Have you run your changes against the testing suite?
- [x] Have you given a written explanation of the change?
- [x] Have you asked a teammate for review?

## Anything else important?

I should probably also update the README with installation instructions as a part of this PR, but I'm super busy rn and that can be done later

### CODE REVIEW

File: src/api_recievers/influxqueries.js [hunk 1]
```diff
@@ -28,7 +93,7 @@ const appData = async (source, appName, currentSession) =>{
```
Comment (stephzhan) [inline src/api_recievers/influxqueries.js:93]: is appData supposed to be commented out?

File: src/api_recievers/influxqueries.js [hunk 2]
```diff
@@ -28,7 +93,7 @@ const appData = async (source, appName, currentSession) =>{
 
   if(write === 'true' && (prevAppName === undefined || prevAppName !== appName)) {
     log.debug("Writing to db");
-    let writeClient = client.getWriteApi(org, bucket, 's');
+    let writeClient = influxClient.getWriteApi(org, influxBuckets.apps, 's');
```
Comment (stephzhan) [inline src/api_recievers/influxqueries.js:96]: additionally, if you pass influxBuckets.apps into getWriteApi, will we have to modify this later to write to the other buckets for tasks

**Review by stephzhan — COMMENTED** [review]
Looks good

**Review by WilsonPrime — APPROVED** [review]
looks good to me

**Pull request merged**